packages/components/molecules/f-navigation-links/src/components/_tests/NavigationLinks.test.js

```diff
@@ -67,4 +67,60 @@ describe('NavigationLinks', () => {
         const links = wrapper.find('[data-test-id="navigationLinks"]');
         expect(links).toMatchSnapshot();
     });
+
+    it('should pass href attribute to `<v-link>` when present', () => {
+        const expected = '/account/info';
+        const wrapper = shallowMount(NavigationLinks, {
+            propsData: {
+                links: [
+                    {
+                        id: 'link1',
+                        to: expected,
+                        name: 'Your account'
+                    }]
+            }
+        });
+
+        const link = wrapper.find('[data-test-id="link1"]');
+
+        expect(link.attributes('to')).toEqual(expected);
```

Comment:
I'm wondering if this is working as expected, because when rendered correctly, `router-link` will still appear in the DOM as an `<a href="/account/info">`.

Previously I've had to [stub the router link component](https://vue-test-utils.vuejs.org/api/components/routerlinkstub.html).

Reply:
Yes but that is within `f-link` itself here its checking the prop/attribute on the stub?

Reply:
If that's the case then cool, was just checking 👍🏻

Reply:
Brilliant thanks